Upstream: Dead.
Source: philantrop@exherbo.org
Reason: Respect the user's CFLAGS.

diff -Naur xmlstarlet-1.0.1.orig/configure.in xmlstarlet-1.0.1/configure.in
--- xmlstarlet-1.0.1.orig/configure.in	2005-03-14 02:30:42.000000000 +0100
+++ xmlstarlet-1.0.1/configure.in	2008-12-25 23:51:33.000000000 +0100
@@ -8,13 +8,12 @@
 AC_CANONICAL_HOST
 AC_PROG_CC
 
-CFLAGS="-Wall -g -pedantic"
 EXTRA_LIBS=
 
 case "${host}" in
   *-hpux* )
       if test "$GCC" != "yes"; then
-        CFLAGS="-Ae"
+        CFLAGS+="-Ae"
       fi
       ;;
   * )
